scanpy/external/tl/_harmony_timeseries.py

```diff
@@ -0,0 +1,462 @@
+"""Harmony time series for data visualization with augmented affinity matrix at discrete timepoints
+"""
+
+from scanpy import logging as logg
+import pandas as pd
+from collections import OrderedDict
+from itertools import chain
+import numpy as np
+import os
```

Comment:
Please sort the imports into 3 groups: Python stdlib, 3rd party modules (numpy, pandas, anndata), then internal scanpy modules as relative import.